tests_and_analysis/test/euphonic_test/test_crystal.py

```diff
@@ -383,7 +383,7 @@ def test_cell_volume(self, crystal, expected_vol):
         (get_crystal('LZO'), lzo_spglib_cell)])
     def test_to_spglib_cell(self, crystal, expected_spglib_cell):
         spglib_cell = crystal.to_spglib_cell()
-        assert type(spglib_cell) is tuple
+        assert isinstance(spglib_cell, tuple)
```

Comment:
Does it have to be a `tuple` or would any collection do? Or is it actually testing it's not `None`?

Reply:
The spglib docs say tuple. Other things might work, but tuple is the official spec. https://spglib.readthedocs.io/en/stable/python-interface.html#crystal-structure-cell

Reply:
It's more what's being tested here. Why are we doing this assert?

Reply:
The job of this method is to produce correct input for spglib. This is a unit test which checks it produces what we understand to be correct input. Part of that spec is the type, part of that spec is the data.

Reply:
Ah! I missed this was in testing. No worries, then.